plugins/info/include/plugins/info/info.h

```diff
@@ -81,15 +81,22 @@ class Info : public PluginBase {
         char product_name[32]; /**< @brief Name of product. */
     };
 
+    /**
+     * @brief Type containing identification.
+     */
+    struct Identification {
+        uint8_t hardware_uuid[18]; /**< @brief UUID of hardware. */
+    };
+
     /**
      * @brief Gets the UUID of the system.
      *
      * If possible this will be a unique identifier provided by hardware.
      *
      * @return a pair containing the result of the request and if successful,
-     * the UUID of the system.
+     * the identification information of the system.
      */
-    std::pair<Result, uint64_t> uuid() const;
+    std::pair<Result, Identification> get_identification() const;
```

Comment:
As a user, how do I know whether a uid2 or a uid1 has been read?
If we do not want to provide a method to access the old version, could we document this here?
Maybe we changing the comment to mention uid and uid2 is enough.

Reply:
Yes I would say we don't support the old version. And we could document it.

Reply:
Right now it just falls back to the old `uid`, is that a problem?

Reply:
It means that the id will silently change after an upgrade from e.g. PX4 stable to master which is suboptimal.
Maybe we can just remove the fallback and call the function get_uid2(). What do you think?

Reply:
Ok removing the fallback makes sense. For the naming, we try to abstract the underlying MAVLink API, so therefore just UID. But I'll mention it in the comment.